scripts/install/opt/seagate/cortx/motr/libexec/m0reportbug-bundler

```diff
@@ -116,27 +124,27 @@ while true; do
          --binlogs)
             #should only accept either true or false
             rc=$(echo $2 | sed -e 's/,/\n/g') ;
-            if [[ "$rc" != "true" &&  "$rc" != "false" ]]; then
+            if [[ "$rc" != "true" && "$rc" != "True" && "$rc" != "false" && "$rc" != "False" ]]; then
```

Comment:
if [ "${rc,,}" != "true" ]; 

similar changes everywhere else can be applied

Reply:
incorporated